docs/trulens/release_blog_1dot.md

```diff
@@ -0,0 +1,242 @@
+
+## TLDR:
+
+1. **Split off trulens-eval from trulens-explain**, and let trulens-eval take over the trulens package name. TruLens-eval is now renamed to TruLens and sits at the root of the TruLens repo, while TruLens-explain has been moved to its own repository, and is installable at trulens-explain.
+
+2. **Separate TruLens-Eval into different trulens packages**
+trulens-core holds core abstractions for database operations, app instrumentation, guardrails and evaluation
+trulens-dashboard gives you the required capabilities to run and operate the TruLens dashboard
+trulens-instrument-* describes a set of third-party integrations for instrumentation and logging, including our popular TruChain and TruLlama offerings that seamlessly instrument LangChain and Llama-Index apps. Each of these integrations can be installed separately as a standalone package, and include trulens-instrument-langchain, trulens-instrument-llamaindex and trulens-instrument-nemo.
+trulens-feedback gives you access to out of the box feedback functions required for running feedback functions. Feedback function implementations must be combined with a selected provider integration.
+trulens-providers-* describes a set of third party integrations for running feedback functions. Today, we offer an extensive set of integrations that allow you to run feedback functions on top of virtually any LLM. These integrations can be installed as standalone packages, and include: trulens-providers-openai, trulens-providers-huggingface, trulens-providers-litellm, trulens-providers-langchain, trulens-providers-bedrock, trulens-providers-cortex.
+
+## Motivation
+
+The driving motivation behind these changes are twofold:
+
+* Minimize the overhead required for TruLens developers to use the capabilities they need
+* Make it easy to understand what code and set of dependencies is needed
+
+## A note on versioning and backwards compatibility:
+
+Today, we’re releasing trulens, trulens-core, trulens-dashboard, trulens-feedback, trulens-providers packages and trulens-instrument packages at v1.0. We will not make breaking changes in the future without bumping the major version.
+
+The base install of trulens will install trulens-core, trulens-feedback and trulens-dashboard making it easy for developers to try TruLens.
+
+trulens-eval will remain at major version 0. All existing trulens-eval imports are mapped to v1 packages, and v1 packages are installable with trulens-eval.
```

Comment:
This bit is confusing. Perhaps rephrase: `trulens-eval` starting 1.0.0 will consist solely of backwards compatibility modules that alias `trulens-*` packages while emitting deprecation warnings. `trulens-eval` version will track `trulens-core` version for 3 months after which the deprecation warnings will be replaced by error messages instructing the user to update to `trulens`.

Reply:
updated